core/src/main/java/com/scalar/db/api/GetBuilder.java

```diff
@@ -79,6 +80,11 @@ public Buildable projections(Collection<String> projections) {
       return this;
     }
 
+    @Override
+    public Buildable projections(String... projections) {
```

Comment:
Is it ok not to add unit tests for the changes to the builders?

Reply:
No, I should add unit tests. I will do that. Thank you!

Reply:
Added unit tests in the latest commit.